src/core.py

```diff
@@ -132,33 +131,17 @@ def _move_modules(self, temp_repo, destination):
         :param string temp_repo: the folder containing the code.
         :param string destination: the folder where the add-on should end up at.
         """
-        folders = self._get_module_folders(temp_repo)
-        for folder in folders:
+        for folder in self._iter_included_modules(temp_repo):
             force_move(folder, destination)
 
-    def _get_module_folders(self, temp_repo):
-        """Get a list of module paths contained in a temp directory.
+    def _iter_included_modules(self, temp_repo):
+        for path in self._iter_modules(temp_repo):
+            module_name = path.split("/")[-1]
+            if self._is_module_included(module_name):
+                yield path
 
-        :param string temp_repo: the folder containing the modules.
-        """
-        paths = (
-            os.path.join(temp_repo, path) for path in os.listdir(temp_repo)
-            if self._is_odoo_module(temp_repo, path) and self._is_module_included(path)
-        )
-        return (path for path in paths if os.path.isdir(path))
-
-    @staticmethod
-    def _is_odoo_module(repo_path, file_name):
-        """Evaluate if the given file/folder is an odoo module.
-
-        :param string module: the name of the file/folder
-        :rtype: bool
-        """
-        file_path = os.path.join(repo_path, file_name)
-        return (
-            os.path.isdir(file_path) and
-            '__manifest__.py' in os.listdir(file_path)
-        )
+    def _iter_modules(self, temp_repo):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/Numigi/gitoo/pullRequest?prid=6195482)